docs/application/native/guides/compliations/watchface-complication-provider.md

```diff
@@ -1,30 +1,32 @@
-# Watchface complication provider
+# Watchface Complication Provider
 
 The watchface complication provider application is a service application to provide complication data.
 Complication provider application provides only data. The watchface application decides how to display the complications.
-The Complication Provider app provides the complication data by using the watchface complication provider api to the watchface app.
+The complication provider application provides the complication data by using the watchface complication provider api to the watchface application.
  
-# Prerequisites
+## Prerequisites
 
-1. To use the Watchface Complication API to communicate with the watchface app, the application has to request permission by adding 
-the following privilege to the tizen-manifest.xml file.
+To enable your application to use the watchface complication functionality:
 
-```xml
-<privileges>
-	<privilege>http://tizen.org/privilege/datasharing</privilege>
-</privileges>
-```
+1. To use the Watchface Complication API to communicate with the watchface application, this application has to request permission by adding
+the following privilege to the `tizen-manifest.xml` file:
 
-2. To use the functions and data types for the Watchface Complication Provider API, include the <watchface-complication-provider.h>.
+    ```xml
+    <privileges>
+    	<privilege>http://tizen.org/privilege/datasharing</privilege>
+    </privileges>
+    ```
 
-```cpp
-#include <watchface-complication-provider.h>
-```
+2. To use the functions and data types for the Watchface Complication Provider API, include the `<watchface-complication-provider.h>` header file in your application:
 
-# Complication Data Update
+    ```cpp
+    #include <watchface-complication-provider.h>
+    ```
 
-When the request occurs to update the complication data, complication provider app can receive the update request by adding 
-watchface_complication_provider_update_requested_cb(). The watchface app can send the update reqeust and the request also occurs after the 
+## Updating the Complication Data
+
+When the request occurs to update the complication data, complication provider app can receive the request by adding
+`watchface_complication_provider_update_requested_cb()`. The watchface application can send the update reqeust and the request also occurs after the

```

Comment:
Typo error "reqeust "

And also change " and this request also occurs after the"